protoc-gen-twirp_ruby/main_test.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestModules(t *testing.T) {
+	tests := []struct {
+		pkgName string
+		modules []string
+	}{
+		{"example", []string{"Example"}},
+		{"twitch.twirp.example.helloworld", []string{"Twitch", "Twirp", "Example", "Helloworld"}},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.pkgName, func(t *testing.T) {
+			modules := Modules(tt.pkgName)
+			if got, want := modules, tt.modules; !reflect.DeepEqual(got, want) {
+				t.Errorf("expected %v; got %v", want, got)
+			}
+		})
+	}
```

Comment:
Thanks for unit testing this! Maybe it is worth adding a case for empty string `""`, package components with 1 letter `"m.v.p"` and invalid packages `"foo.....bar"`.